README.md

```diff
@@ -1,70 +1,220 @@
 # @daml/hub-react
 
-> React functions for Daml applications running on [projectDABL](https://projectdabl.com/).
+> React functions for Daml applications running on [Daml Hub](https://hub.daml.com/).
 
 Copyright 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All Rights Reserved. SPDX-License-Identifier: Apache-2.0
 
 ## Documentation
 
 This library aims to complement [@daml/react](`https://www.npmjs.com/package/@daml/react),
-to make it easier to write [React](https://reactjs.org/) applications on DABL specifically.
-To that end we provide for convenience.
+to make it easier to write [React](https://reactjs.org/) applications on Daml Hub specifically.
 
 This library exports Typescript declarations, and should work in a Typescript project out of the box.
 
+See the [DA Marketplace](https://github.com/digital-asset/da-marketplace) app as a reference for more advanced usage of this library.
+
 ## Usage
 
-### Well Known End point
+### Login
 
-DABL provides an `.well-known` endpoint ([see](https://docs.projectdabl.com/api/iam/#listening-for-new-users)), that provides the identity of Parties who can run automation or
-observe common information for your application.
+Any app that runs on Daml Hub needs to authenticate to the ledger. Authentication is done via stateless access tokens that are signed and distributed by Daml Hub, provided in the JSON Web Token format. These tokens expire every 24 hours.
 
-```typescript
-// Within a
-<WellKnownPartiesProvider>
-  ...
-  // This hook will provide well known parties from DABL, as well as status information regarding the request.
-  let { parties, loading, error } = useWellKnownParties();
+Typical authentication involves forwarding the user through an interactive sign-in flow that redirects back to the application, with the user's access token made available to the app via the `DAMLHUB_LEDGER_ACCESS_TOKEN` cookie.
```

Comment:
FWIW a more active voice might make this sentence a bit more clear.. like:
`Typically, authentication forwards the user through an interactive sign-in flow then redirects back to the application. The user's access token is then made available to the app via the `DAMLHUB_LEDGER_ACCESS_TOKEN` cookie.`

Reply:
Thanks, that's a nice improvement!